src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -307,6 +388,7 @@ private String buildOAuthRedirectUrl() {
      * This is where the user comes back to at the end of the OpenID redirect ping-pong.
      */
     public HttpResponse doFinishLogin(StaplerRequest request) throws IOException {
+    	LOGGER.fine("Realm do finish " + request);
```

Comment:
Are logging messages supposed to prefix by the method that is calling them?  It's inconsistently applied in your debugging.